tests/python/accuracy/public_scope.json

```diff
@@ -100,24 +100,24 @@
                     "(50, 279, 368, 385, 0.915, 18, sheep)"
                 ]
             }
-        ],
-        "TODO: there is a bug in openvino. Tickets 106890 and 106891": {
-            "name": "otx_models/det_mobilenetv2_atss_bccd.xml",
-            "type": "DetectionModel",
-            "test_data": [
-                {
-                    "image": "BloodImage_00007.jpg",
-                    "reference": [
-                        "(494, 159, 637, 308, 0.700, 2, WBC)",
-                        "(28, 139, 136, 228, 0.623, 1, RBC)",
-                        "(535, 374, 638, 479, 0.535, 1, RBC)",
-                        "(513, 8, 633, 151, 0.454, 1, RBC)",
-                        "(197, 86, 412, 286, 0.405, 1, RBC)",
-                        "(21, 291, 143, 399, 0.401, 1, RBC)"
-                    ]
-                }
-            ]
-        }
+        ]
+    },
```

Comment:
Any idea what caused the difference?

Reply:
There was a nested test entry marked as TODO. Now I moved it out of one layer of nesting to outside to allow running it